cmd/paths.go

```diff
@@ -266,39 +266,85 @@ func pathsUpdateCmd(a *appState) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "update path_name",
 		Aliases: []string{"n"},
-		Short:   `Update a path such as the filter rule ("allowlist", "denylist", or "" for no filtering) and channels`,
+		Short:   `Update a path such as the filter rule ("allowlist", "denylist", or "" for no filtering), channels, `,
```

Comment:
Looks like you maybe intended to add more words after the `,` at the end of this line

Reply:
yes I did, good catch